test/cert-ceremonies/root-ceremony-rsa.yaml

```diff
@@ -20,3 +20,14 @@ certificate-profile:
     key-usages:
         - Cert Sign
         - CRL Sign
+skip-lints:
+   - e_ext_authority_key_identifier_missing
+   - e_ext_authority_key_identifier_no_key_identifier
+   - e_sub_ca_aia_does_not_contain_ocsp_url
+   - e_sub_ca_aia_missing
+   - e_sub_ca_certificate_policies_missing
+   - e_sub_ca_crl_distribution_points_missing
+   - n_ca_digital_signature_not_set
+   - n_mp_allowed_eku
+   - n_sub_ca_eku_missing
+   - w_sub_ca_aia_does_not_contain_issuing_ca_url
```

Comment:
This is the set of lints which fail when `startservers.py` and `test/cert-ceremonites/generate.go` use the ceremony tooling to generate a new root.

The good news is that no lints fail when those files generate an intermediate!